packages/strapi-plugin-users-permissions/controllers/UsersPermissions.js

```diff
@@ -64,13 +64,14 @@ module.exports = {
 
   async getPermissions(ctx) {
     try {
-      const { lang } = ctx.query;
-      const plugins = await strapi.plugins[
-        'users-permissions'
-      ].services.userspermissions.getPlugins(lang);
+      // @alexandrebodin I am removing this as it is not needed anymore
```

Comment:
@alexandrebodin can you approve that this is not needed anymore?

Reply:
If you are not using the information anymore you can delete this yeas. You can also cleanup the `getActions` method in the service and remove the `withInfo` and the `information` property in the returned data then